index.html

```diff
@@ -90,7 +90,7 @@ <h2 class="support-text-2 default-custom-fonts lets-text-font">LETS CONNECT</h2>
           <li><img class="counter" src="images/Counter.svg" alt="Counter"></li>
```

Comment:
- [ ] Fix margin/padding according to the Figma design you selected.
- [ ] Fix padding for "Get my resume" `button` tag.
- [ ] [OPTIONAL] Fix HTML image in the "Languages" section.

Figma Design | Yours

![image](https://user-images.githubusercontent.com/60164239/129461310-62d76e8d-dbca-4680-830e-686b50cf4f03.png)